src/clojure/com/interrupt/ibgateway/component/ewrapper_impl.clj

```diff
@@ -138,7 +159,8 @@
 (def default-chs-map
   {:publisher (-> 1000 async/sliding-buffer async/chan)
    :account-summary-ch acct-summary/account-summary-ch
-   :account-updates-ch acct-updates/account-updates-ch})
+   :account-updates-ch acct-updates/account-updates-ch
+   :portfolio-updates-ch portfolio/portfolio-updates-ch})
```

Comment:
Not sure what you mean. Can you give an example?

Reply:
@xiongtx Sure. You'll notice [in master](https://github.com/twashing/ibgateway/blob/master/src/clojure/com/interrupt/ibgateway/component/ewrapper_impl.clj#L174), that I'm not using `(default-chs-map :publisher)`.  That's because that channel needs to be *"reloadable"*. So if I close the channel, restarting all the components gets me a fresh channel. That's all.

And I'm sure we'll want the same pattern for the other channels.